site/blog/WIIFM

```diff
@@ -1,23 +1,22 @@
+1. The Learner: The Learner is interested in personal development and for someone who wants to learn more about some of the unique requirements for the network edge and wants to pick up some of the cool technology that’s being deployed there, StarlingX is a great training ground. Operational Technology (OT) is different than IT. For example, each has different latencies, bandwidth and scale requirements - among other things. StarlingX helps to bridge the gap through a Kubernetes container environment. If learning by doing is your thing, and you want to learn how to implement cutting edge technology in some of the world’s most demanding environments, there is a place in the StarlingX “classroom” for you.
 
-#2. The Self-Delegator: This is a person whose company has evaluated StarlingX and determined it could be a good fit for their needs…but there is a critical feature that doesn’t exist yet. By getting involved in StarlingX the Self-Delegator can have a voice in the discussion on how to shape the 5G edge, and has the opportunity to advocate for the features that they think will be required to effectively move the industry forward. But they not only advocate, they take a pro-active role in getting it done – supported by a team of developers motivated to produce the edge infrastructure for the future. This is for the person who goes to their boss and says, “I think we should do XYZ” and their boss says, “I agree. When can you start on that?” and they get excited. You don’t just get a seat at the table, you get an apron and a spot in the kitchen to determine what’s served.
+2. The Self-Delegator: This is a person whose company has evaluated StarlingX and determined it could be a good fit for their needs…but there is a critical feature that doesn’t exist yet. By getting involved in StarlingX the Self-Delegator can have a voice in the discussion on how to shape the 5G edge, and has the opportunity to advocate for the features that they think will be required to effectively move the industry forward. But they not only advocate, they take a pro-active role in getting it done – supported by a team of developers motivated to produce the edge infrastructure for the future. This is for the person who goes to their boss and says, “I think we should do XYZ” and their boss says, “I agree. When can you start on that?” and they get excited. You don’t just get a seat at the table, you get an apron and a spot in the kitchen to determine what’s served.
```

Comment:
I would delete: 'This is for the person who goes to their boss and says, “I think we should do XYZ” and their boss says, “I agree. When can you start on that?” and they get excited.' as I think it might demotivate people as especially in larger companies that's not necessarily plays out like this. And it would make the text a little shorter which makes it an easier read.

Reply:
OK. I've edited that paragraph a bit. Thanks